cli/build.rs

```diff
@@ -64,29 +66,33 @@ fn create_compiler_snapshot(
   );
   runtime_isolate.register_op(
     "op_fetch_asset",
-    deno_typescript::op_fetch_asset(custom_libs),
+    op_fetch_asset::op_fetch_asset(custom_libs),
   );
 
-  js_check(
-    runtime_isolate.execute("typescript.js", deno_typescript::TYPESCRIPT_CODE),
-  );
+  js_check(runtime_isolate.execute(
+    "typescript.js",
+    &std::fs::read_to_string("typescript/lib/typescript.js").unwrap(),
+  ));
 
   create_snapshot(runtime_isolate, snapshot_path, files);
 }
 
+fn ts_version() -> String {
+  let data = include_str!("typescript/package.json");
+  let pkg: serde_json::Value = serde_json::from_str(data).unwrap();
+  pkg["version"].as_str().unwrap().to_string()
+}
+
```

Comment:
IMO this is redundant and could remove build depdendency on `serde` and `serde_json`, why not just hard code it? TS update process it manual anyway

Reply:
We will forget it.  I would leave it for now until we have a better approach overall to managing TypeScript (which I still think a seperate crate is the right way for the compiler.

The other way would be to return it when creating compiler snapshot as an op out with the `ts.version` information.  That is actually more reliable.  Again, something I think should be done in the compiler crate.